api/api.go

```diff
@@ -66,6 +69,51 @@ type CreateOrbResponse struct {
 	GQLResponseErrors
 }
 
+// Orb is a struct for containing the yaml-unmarshaled contents of an orb
+type Orb struct {
+	Commands  map[string]struct{}
+	Jobs      map[string]struct{}
+	Executors map[string]struct{}
+}
+
+func addOrbElementsToBuffer(buf *bytes.Buffer, name string, elems map[string]struct{}) error {
+	var err error
+
+	if len(elems) > 0 {
+		_, err = buf.WriteString(fmt.Sprintf("  %s:\n", name))
+		if err != nil {
```

Comment:
It's safe to ignore the error from `bytes.Buffer.Write` methods. They can't actually error, they have an error return so that they fit the `io.Writer` interface.

With the error removed the FIXME/panic in the caller should be save to remove as well.